### ISSUE TITLE
Templates worked for me after one change...

### ISSUE THREAD

**Issue by Stephen-McDaniel** [JohnCoene/ambiorix.generator#1]
Before import("views"), I needed to add a working directory at root of template path:

<pre>
# app.R is in /myapps/template01
setwd("/myapps/template01")
</pre>

Then it worked.  Without this, I had this error:

<pre>
rror: [ENOENT] Failed to search directory '//views': no such file or directory
</pre>

Best,
Stephen

**Comment by JohnCoene** [comment]
I don't think you should be in the `/template` directory to call `import`, this might be a bug.
